.github/workflows/release.yml

```diff
@@ -37,7 +22,7 @@ jobs:
       version: ${{ needs.prepare-version.outputs.version }}
```

Comment:
^ remove these 2 lines since `prepare-version` is gone

Reply:
whoops good catch, fixed